support/diagnostics/eck-dump.sh

```diff
@@ -34,6 +34,28 @@ RESOURCES_NS="default"
 OUTPUT_DIR=""
 ZIP=""
 VERBOSE=""
+# go-template to dump metadata in textual form excluding sensitive fields like "last-applied-configuration"
+METADATA_TPL=$(cat <<'EOF'
+{{range .items}}
+{{"---" -}}
+{{range $k,$v := .metadata }}
+{{ printf `"%v": ` $k -}}
+  {{- if eq $k "annotations" "labels" -}}
+    {{- $nth := false -}}
+    {{- range $k,$v := . -}}
+      {{- if ne $k "kubectl.kubernetes.io/last-applied-configuration" -}}
+        {{- if $nth -}}{{- "," -}}{{- end -}}
+        {{- printf `"%v":"%v"` $k $v -}}
+        {{$nth = true}}
+      {{- end -}}
+    {{- end -}}
+  {{else}}
+    {{- printf `"%v"` $v -}}
```

Comment:
Nit: `%v` vs `$v` doesn't help to decypher the template:
```suggestion
{{range $key,$value := .metadata }}
{{ printf `"%v": ` $key -}}
  {{- if eq $key "annotations" "labels" -}}
    {{- $nth := false -}}
    {{- range $key,$value := . -}}
      {{- if ne $key "kubectl.kubernetes.io/last-applied-configuration" -}}
        {{- if $nth -}}{{- "," -}}{{- end -}}
        {{- printf `"%v":"%v"` $key $value -}}
        {{$nth = true}}
      {{- end -}}
    {{- end -}}
  {{else}}
    {{- printf `"%v"` $value -}}
```